packages/desktop-client/src/components/mobile/transactions/TransactionEdit.jsx

```diff
@@ -161,7 +163,7 @@ export function Status({ status, isSplit }) {
         textAlign: 'left',
       }}
     >
-      {titleFirst(status) + (isSplit ? ' (Split)' : '')}
+      {titleFirst(status) + (isSplit ? t(' (Split)') : '')}
```

Comment:
Maybe `t('{title} (Split)')` might be clearer to translators?

Reply:
good shout: 586012f47098568620fa7ff754cbc0fa574a815c